HW_1.py

```diff
@@ -0,0 +1,346 @@
+# coding=utf-8
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+def print_hi(name):
+    # Use a breakpoint in the code line below to debug your script.
+    print("Hi, {0}".format(name))  # Press Ctrl+F8 to toggle the breakpoint.
+
+
+# Press the green button in the gutter to run the script.
+if __name__ == '__main__':
+    print_hi('PyCharm')
+
+# See PyCharm help at https://www.jetbrains.com/help/pycharm/
+
+###
+for countdown in 5,4,3,2,1, "hey!":
+    print (countdown)
+
+
+
+# 1 line: Output
+print ('Hello, world!')
+print ("Hello, world!")
+
+
+# 2 lines: Input, assignment - Нужно ли ответ ВСЕГДА писать в кавычках?
+name = input('What is your name?\n')
+print ('Hi, %s.' % name)
+
+
+# 3 lines: For loop, built-in enumerate function, new style formatting
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print ("iteration {iteration} is {name}".format(iteration=i, name=name))
+
+# 4
+parents, babies = (1, 1)
+while babies < 100:
+    print ('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+
+# 5 lines: Functions
+def greet(name):
+    print ('Hello', name)
+
+greet('Jack')
+greet('Jill')
+greet('Bob')
+
+# 6 регулярные выражения (модуль re - regex) https://pythonru.com/primery/primery-primeneniya-regulyarnyh-vyrazheniy-v-python
+import re
+for test_string in ['555-1212', 'ILL-EGAL']:
+    if re.match(r'^\d{3}-\d{4}$', test_string):
+        print (test_string, 'is a valid US local phone number')
+    else:
+        print (test_string, 'rejected')
+
+# 7

```

Comment:
you have 2 lines between task 1 and task 2. Here you have 1 blank line. Styling should be consistent.